src/content/en/2019/performance.md

```diff
@@ -0,0 +1,266 @@
+
+## The state of performance
+
+The [HTTP Archive](https://httparchive.org/) is an entirely lab-based tool, so to get field data we need to make use of a different dataset. In this section we're using the [Chrome UX Report](http://bit.ly/chrome-ux-report) (CrUX), a public dataset from Google that consists of all the same websites as the HTTP Archive and aggregates how Chrome users actually experience them. Experiences are categorized by:
+
+- the form factor of the users' devices
+  - desktop
+  - phone
+  - tablet
+- users' effective connection type (ECT) in mobile terms
+  - offline
+  - slow 2G
+  - 2G
+  - 3G
+  - 4G
+- users' geographic location
+
+Experiences are measured monthly including paint, load, and interactivity metrics. The first metric we'll look at is [First Contentful Paint](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint) (FCP). This is the time users spend waiting for the page to display something useful to the screen, like an image or text. Next, we'll look at look at a loading metric, [Time to First Byte](https://web.dev/time-to-first-byte) (TTFB). This is a measure of how long the web page took from the time of the user's navigation until they received the first byte of the response. And finally, the last field metric we'll look at is [First Input Delay](https://developers.google.com/web/updates/2018/05/first-input-delay) (FID). This is a relatively new metric and one that represents parts of the UX other than loading performance. It measures the time from a user's first interaction with a page's UI until the time the browser's main thread is ready to process the event.
+
+So let's dive in and see what kind of insights we can find.
+
+### FCP
+
+<figure>
+// Chart: flame distribution of 07_01
+<figcaption>Figure 1. Distribution of websites' fast, average, and slow FCP performance.</figcaption>
+</figure>
+
+In Figure 1 above you can see how FCP experiences are distributed across the web. Out of the millions of websites in the CrUX dataset, this chart compresses the distribution down to 1,000 websites where each vertical slice represents a single website. The chart is sorted by the percent of fast FCP experiences, which are those occurring in less than 1 second. Slow experiences occur in 2.5 seconds or more, and average experiences are everything in between. At the extremes of the chart, there are some websites with almost 100% fast experiences and some websites with almost 100% slow experiences. In between, websites have a combination of fast, average, and slow performance that seems to lean more towards fast or average than slow, which is good.
```

Comment:
Adding an image like this would be really helpful to readers. To let them see how all these different metrics relate, visually. Taken from: https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics
![perf-metrics-load-timeline](https://user-images.githubusercontent.com/684747/67230609-343cf480-f403-11e9-9877-dc6852e71693.png)

Reply:
ACK. I have some gripes with that image and would prefer to use something tailored to the metrics we're discussing in this chapter, but unfortunately I don't have time to create something like that. The FCP link above does link to that page, if that helps.

Reply:
Yeah I only meant to use this image as an example, not a drop in for the chapter. A custom visual would definitely be needed. Seeing as we don't have time for that though, I'll close this out.